snuba/datasets/outcomes.py

```diff
@@ -85,6 +114,7 @@ def __init__(self):
             order_by="(org_id, project_id, timestamp)",
             partition_by="(toMonday(timestamp))",
             settings={"index_granularity": 16384},
+            migration_function=outcomes_write_migrations,
```

Comment:
I don't think we ever ran the migration script on a materialized view that has two schemas inside. Could you please remove the columns locally, then run snuba migration --dataset outcomes to ensure it works properly ?

Reply:
Running the migrations locally produces the following: ```
Executing migration: ALTER TABLE outcomes_hourly_local ADD COLUMN bytes_received Nullable(UInt64)
Executing migration: ALTER TABLE outcomes_raw_local ADD COLUMN size Nullable(UInt32)```

Reply:
I had to run bootstrap to get `bytes_received` added to `outcomes_mv_hourly_local`

Reply:
aren't the two statements contradictory ?
if it ran `Executing migration: ALTER TABLE outcomes_hourly_local ADD COLUMN bytes_received Nullable(UInt64)`, why did you also have to run bootstrap ?

Reply:
I had to drop the table outcomes_mv_hourly_local and run bootstrap to get it to be recreated with the `bytes_received` column. I ran the migration afterwards

Reply:
So it means that when you saw 
```
Executing migration: ALTER TABLE outcomes_hourly_local ADD COLUMN bytes_received Nullable(UInt64)
```
the statement was not actually executed, right ? 
This is an issue, because we need to provide a migration way that does not require the user to drop the table.
Did you ever try to run the migration on the table with the old format and check if it actually worked ?

Reply:
Yes- I ran the migration prior to bootstrapping and it did add those two columns. I only ran bootstrap after dropping `outcomes_mv_hourly_local` to have it recreated with the additional `bytes_received` column.